cont/base/springcontent/LuaHandler/Utilities/actions.lua

```diff
@@ -13,258 +13,276 @@
 --
 --  Removals
 --
 
 local function RemoveAddonAction(addon, cmd, types, _)
-	assert(_ == nil, "actionHandler:Foobar() is deprecated, use actionHandler.Foobar()!")
-
-	-- default to removing all
-	local text, keyPress, keyRepeat, keyRelease = ParseTypes(types, "tpRr")
-
-	local tSuccess, pSuccess, RSuccess, rSuccess = false, false, false, false
-
-	if (text)       then tSuccess = RemoveAction(textActions, addon, cmd)       end
-	if (keyPress)   then pSuccess = RemoveAction(keyPressActions, addon, cmd)   end
-	if (keyRepeat)  then RSuccess = RemoveAction(keyRepeatActions, addon, cmd)  end
-	if (keyRelease) then rSuccess = RemoveAction(keyReleaseActions, addon, cmd) end
-
-	return tSuccess, pSuccess, RSuccess, rSuccess
+  assert(_ == nil, "actionHandler:Foobar() is deprecated, use actionHandler.Foobar()!")
+
+  -- default to removing all
+  local text, keyPress, keyRepeat, keyRelease = ParseTypes(types, "tpRr")
+
+  local tSuccess, pSuccess, RSuccess, rSuccess = false, false, false, false
+
+  if text then
+    tSuccess = RemoveAction(textActions, addon, cmd)
+  end
+  if keyPress then
+    pSuccess = RemoveAction(keyPressActions, addon, cmd)
+  end
+  if keyRepeat then
+    RSuccess = RemoveAction(keyRepeatActions, addon, cmd)
+  end
+  if keyRelease then
+    rSuccess = RemoveAction(keyReleaseActions, addon, cmd)
+  end
+
+  return tSuccess, pSuccess, RSuccess, rSuccess
 end
 
-
 local function RemoveAddonActions(addon, _)
-	assert(_ == nil, "actionHandler:Foobar() is deprecated, use actionHandler.Foobar()!")
+  assert(_ == nil, "actionHandler:Foobar() is deprecated, use actionHandler.Foobar()!")
 
-	ClearActionList(textActions, addon)
-	ClearActionList(keyPressActions, addon)
-	ClearActionList(keyRepeatActions, addon)
-	ClearActionList(keyReleaseActions, addon)
+  ClearActionList(textActions, addon)
+  ClearActionList(keyPressActions, addon)
+  ClearActionList(keyRepeatActions, addon)
+  ClearActionList(keyReleaseActions, addon)
 end
 
-
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 --
 --  Calls
 --
 
-
 local function KeyAction(press, _, _, isRepeat, _, actions)
-	if (not(actions and next(actions))) then return false end
-
-	local actionSet
-	if (press) then
-		actionSet = isRepeat and keyRepeatActions or keyPressActions
-	else
-		actionSet = keyReleaseActions
-	end
-	for _,bAction in ipairs(actions) do
-		local bCmd, bOpts = next(bAction, nil)
-		local words = MakeWords(bOpts)
-		if (TryAction(actionSet, bCmd, bOpts, words, isRepeat, not press, actions)) then
-			return true
-		end
-	end
-
-	return false
+  if not (actions and next(actions)) then
+    return false
+  end
+
+  local actionSet
+  if press then
+    actionSet = isRepeat and keyRepeatActions or keyPressActions
+  else
+    actionSet = keyReleaseActions
+  end
+
+  for _, action in ipairs(actions) do
```

Comment:
The scope of this block for both these and the corresponding luaui file are the relevant sections